E3db/Classes/Config.swift

```diff
@@ -79,9 +79,9 @@ public struct Config: Codable {
         case apiSecret     = "api_secret"
         case publicKey     = "public_key"
         case privateKey    = "private_key"
-        case baseApiUrl    = "base_api_url"
-        case publicSigKey  = "public_sig_key"
-        case privateSigKey = "private_sig_key"
+        case baseApiUrl    = "api_url"
```

Comment:
ack, this is what @efabens was asking about. It's going to be much more backwards compatible if we don't change these keys here but have the flutter code look for them using their current names and convert to another format with the expected names as needed (but again all on the flutter side, not on the e3db-swift sdk)